README.md

```diff
@@ -75,6 +76,34 @@ following fields:
 * `events`: An event sink which supports the same interface as Node.js's `EventEmitter`. Required. See
   below for the events that get generated.
 
+### `extractMetadata(options)`
+
+This method returns a simple object with metadata inferred from the document.
+The `options` accepted are equal to those in `validate()`, except that a `profile` is not necessary and will be ignored (finding out the profile is one of the
+goals of this method).
+
+The returned `Object` may contain up to 3 properties: `detectedProfile`, `detectedDeliverers` and `detectedDelivererIDs`.
```

Comment:
`detectedDeliverers` and `detectedDeliverersIDs` are redundant and may lead to inconsistencies. I'd say keep `detectedDelivererIDs` only. From the id, the name and the homepage of the group are easy to get using the API